src/constants/links.js

```diff
@@ -25,16 +26,19 @@ export const header = {
     ],
     // second dropdown end
   ],
-  parentLinks: [{ LABEL: 'navigation.parentLinkLabels.0', LINK: 'link-five' }], // parent (standalone) links
+  parentLinks: [
+    { LABEL: 'navigation.parentLinkLabels.0', LINK: 'home' },
+    { LABEL: 'navigation.parentLinkLabels.1', LINK: 'documentation' },
```

Comment:
Do we want "Home" and "Documentation" to be parent links or nav dropdown links? I'm leaning toward parent links because the user is more likely to see them that way. 

Also should we make it so that clicking on the MOCTO logo takes people back to the home page?

Reply:
Parent links, they aren't related. Great idea; clicking the MOCTO logo can take people back to the home page. You can add it to the backlog or go about it in this PR if you'd like.

Reply:
Okay great - I'll add it to this PR